recipes/libsodium/1.0.18/conanfile.py

```diff
@@ -188,6 +188,6 @@ def _get_configure_args(self, absolute_install_dir):
             self._autotools_bool_arg("soname-versions", self.options.use_soname),
             self._autotools_bool_arg("pie", self.options.PIE)
         ]
-        if self.options.fPIC:
+        if hasattr(self, 'options.fPIC') and self.options.fPIC:
```

Comment:
you can also use `self.options.get_safe("fPIC")` documented here: https://docs.conan.io/en/latest/reference/conanfile/attributes.html?highlight=get_safe (works with options as well)